tests/manual-test-cases/Group5-Functional-Tests/5-5-Heterogeneous-ESXi.robot

```diff
@@ -37,7 +37,9 @@ Test
     Append To List  ${list}  ${esx3}
 
     # Finish vCenter deploy
-    ${output}=  Wait For Process  ${pid-vc}
+    ${output}=  Wait For Process  ${pid-vc}  timeout=40 minutes  on_timeout=terminate
```

Comment:
This means that the robot framework could terminate the vc deployment process if it exceeds 40min, so that we can still grab the log bundle gracefully without drone terminating the test, right?